pytorch_lightning/trainer/training_loop.py

```diff
@@ -305,13 +306,13 @@ def has_arg(self, *args):
 
     def train(self):
         # add signal handlers for process kills
-        def _signal_kill_handler(*args):
-            return TrainerTrainLoopMixin.run_training_teardown(self)
-
-        orig_signal_handlers = {}
```

Comment:
cc: @justusschock

Reply:
as mentioned @justusschock i disabled this since it was a blocker.
TBH, i don't know how this was allowed on master with the exceptions being thrown every time

Reply:
I'm fine with temporarily disabling this. In our CI these exceptions did not appear